core-services/prow/02_config/open-cluster-management/prometheus/OWNERS

```diff
@@ -0,0 +1,8 @@
+approvers:
+  - songleo
```

Comment:
Highly recommend having more than one approver. If you're not available, then there will be no way to get PRs merged.